### PR TITLE
Build history

### PR DESCRIPTION
**Pull request by kofj**
Show build history info in the tag detail page as a default table panel.

### CODE REVIEW

File: src/ui_ng/lib/src/tag/tag-detail.component.html [hunk 1]
```diff
@@ -33,23 +46,37 @@ <h2 class="custom-h2">{{repositoryId}}:{{tagDetails.name}}</h2>
...
                     <div class="second-column">
-                        <div><clr-icon shape="error" size="24" class="is-error"></clr-icon> {{highCount}} {{'VULNERABILITY.SEVERITY.HIGH' | translate }}&nbsp;{{'TAG.LEVEL_VULNERABILITIES' | translate }}</div>
-                        <div class="second-row"><clr-icon shape="exclamation-triangle" size="24" class="tip-icon-medium"></clr-icon>{{mediumCount}} {{'VULNERABILITY.SEVERITY.MEDIUM' | translate }}&nbsp;{{'TAG.LEVEL_VULNERABILITIES' | translate }}</div>
-                        <div><clr-icon shape="play" size="20" class="tip-icon-low rotate-90"></clr-icon>{{lowCount}} {{'VULNERABILITY.SEVERITY.LOW' | translate }}&nbsp;{{'TAG.LEVEL_VULNERABILITIES' | translate }}</div>
-                        <div class="second-row"><clr-icon shape="help" size="18"></clr-icon>{{unknownCount}} {{'VULNERABILITY.SEVERITY.UNKNOWN' | translate }}&nbsp;{{'TAG.LEVEL_VULNERABILITIES' | translate }}</div>
+                        <div>
+                            <clr-icon shape="error" size="24" class="is-error"></clr-icon> {{highCount}} {{'VULNERABILITY.SEVERITY.HIGH' | translate }}&nbsp;{{'TAG.LEVEL_VULNERABILITIES'
```
Comment (steven-zou) [inline src/ui_ng/lib/src/tag/tag-detail.component.html]: Seems this part is only code formatting changes. But the format is a little inconsistent with the original style.

File: src/ui_ng/lib/src/tag/tag-detail.component.spec.ts [hunk 2]
```diff
@@ -5,16 +5,17 @@ import { ResultGridComponent } from '../vulnerability-scanning/result-grid.compo
...
 import { SERVICE_CONFIG, IServiceConfig } from '../service.config';
 import { TagService, TagDefaultService, ScanningResultService, ScanningResultDefaultService } from '../service/index';
 import { FilterComponent } from '../filter/index';
 import { VULNERABILITY_SCAN_STATUS } from '../utils';
-import {VULNERABILITY_DIRECTIVES} from "../vulnerability-scanning/index";
-import {LabelPieceComponent} from "../label-piece/label-piece.component";
-import {JobLogViewerComponent} from "../job-log-viewer/job-log-viewer.component";
-import {ChannelService} from "../channel/channel.service";
-import {JobLogService, JobLogDefaultService} from "../service/job-log.service";
+import { VULNERABILITY_DIRECTIVES } from "../vulnerability-scanning/index";
```
Comment (steven-zou) [inline src/ui_ng/lib/src/tag/tag-detail.component.spec.ts]: Good format!

File: src/ui_ng/lib/src/tag/tag-detail.component.html (hunk 1 above)
Comment (kofj) [reply src/ui_ng/lib/src/tag/tag-detail.component.html]: It is an error, will be fixed later.

**Comment by steven-zou** [comment]
@kofj 

Let' holding for a while and merge it into master after @ninjadq and @zhoumeina complete the upgrading work of Clarity and Angular.

Rebase will be required then!

**Comment by steven-zou** [comment]
@kofj 

  The related libraries of front-end UI are upgraded to new versions (Angular to 6.x, clarity to 0.12). Now we can start the merging work of this PR. Before doing that, I think you need to rebase your code to based on the new code base. Once you have done, please let us know.

**Comment by clouderati** [comment]
Hi @kofj – please rebase when you get a moment. Thanks!

**Comment by kofj** [comment]
@steven-zou @ninjadq All checks have passed, review please.

File: src/portal/src/i18n/lang/zh-cn-lang.json [hunk 3]
```diff
@@ -813,32 +816,31 @@
...
-    "START_TIME": "创建时间",
-    "UPDATE_TIME": "更新时间",
-    "LOGS":"日志",
-    "PENDING":"未开始",
-    "FINISHED":"已完成",
-    "STOPPED":"已停止",
-    "RUNNING":"执行中",
-    "ERROR":"错误",
+    "STATUS": "状态",
+    "START_TIME": "开始时间",
```
Comment (zhoumeina) [inline src/portal/src/i18n/lang/zh-cn-lang.json]: why need to modify this update_time to endtime. It should not be modified in this pr.

**Pull request merged**